### ISSUE TITLE
Can't detect sexual chromosome?

### ISSUE THREAD

**Issue by Dolce-99** [ghm17/LOGODetect#21]
Just wanted to confirm -- can X Chromosome be the input of LOGODetect? I checked the R code, it seems like LOGODetect only goes through chromosome 1-22.

**Comment by ghm17** [comment]
Yes, the current software only support analyzing SNPs in the autosomes.
